web-animations-2/Overview.bs

```diff
@@ -262,6 +262,9 @@ Append:
 >              *   Set [=iteration duration=] = [=intrinsic iteration duration=]
 >          Otherwise:
 >              *   Set [=iteration duration=] = |specified iteration duration|
+>
+> An animation's <dfn export for=animation>progress</dfn> is the ratio of its
+>   [=animation/current time=] to its associated effect's [=end time=].
```

Comment:
This needs to refer to the actual definition of the value. I know @fantasai and I talked about this a bit when she fixed up the spec.

When an implementer or developer clicks on a link to `progress` they should be able to get to the algorithm that defines it fairly easily, not just an abstract description of what it is.

I'd suggest moving the definition of `progress` to the same section as the algorithm. If for some reason that's not possible, we should link to the algorithm from here.

Reply:
Thanks for the suggestion. Moving it to the algorithm section seems fine, done.